src/hal/src/image.rs

```diff
@@ -208,29 +208,29 @@ impl AaMode {
 
 /// How to [filter](https://en.wikipedia.org/wiki/Texture_filtering) the
 /// texture when sampling. They correspond to increasing levels of quality,
-/// but also cost. They "layer" on top of each other: it is not possible to
-/// have bilinear filtering without mipmapping, for example.
-///
-/// These names are somewhat poor, in that "bilinear" is really just doing
-/// linear filtering on each axis, and it is only bilinear in the case of 2D
-/// textures. Similarly for trilinear, it is really Quadralinear(?) for 3D
-/// textures. Alas, these names are simple, and match certain intuitions
-/// ingrained by many years of public use of inaccurate terminology.
+/// but also cost.
 #[derive(Clone, Copy, Debug, Eq, Hash, Ord, PartialEq, PartialOrd)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
 pub enum FilterMethod {
```

Comment:
please note that there is a `BlitFilter` type that can now be removed as a duplicate

Reply:
also, we might want to shorten it to just `Filter` as well as map the values directly to Vulkan counterparts for FFI

Reply:
thanks, done